lib/main.dart

```diff
@@ -21,7 +21,7 @@ class MyApp extends StatelessWidget {
         // or simply save your changes to "hot reload" in a Flutter IDE).
         // Notice that the counter didn't reset back to zero; the application
         // is not restarted.
-        primarySwatch: Colors.blue,
+        primarySwatch: Colors.red,
```

Comment:
Why red? It's too ugly!!!

Reply:
I'm just following orders, they told me to change to red